system/config/templates/features/aca_individual_market/financial_assistance.yml

```diff
@@ -18,3 +18,9 @@ registry:
             item: false
           - key: :ivl_application_checklist
             item: 'IVL_Application_Checklist_Final_02172021.pdf'
+      - key: :indian_health_service_question
```

Comment:
This should be the file we're using for all financial assistance stuff.

Reply:
@ZASMan I need to rebase my branch